doc/Language/exceptions.pod6

```diff
@@ -84,9 +93,15 @@ methods apply to C<$_>, which holds the C<Exception> within the C<CATCH> block.
          }
     }
 
-Note that the match target is a role. To allow user defined exceptions to match
-in the same manner, they must implement the given role. Just existing in the
-same namespace will look alike but won't match in a C<CATCH> block.
+Note that the match target is a role. To allow user defined exceptions to
+match in the same manner, they must implement the given role. Just existing
+in the same namespace will look alike but won't match in a C<CATCH> block.
```

Comment:
I don't quite understand this sentence. It was there prior to this PR, but as long as you're editing it, some nits
1. Not clear what's that precise role you're mentioning.
2. The second sentence is difficult to parse. It refers to the existence of what? Will look like what?